src/mxf_helper/RDD36MXFDescriptorHelper.cpp

```diff
@@ -50,54 +50,44 @@ typedef struct
     uint32_t width;
     uint32_t height;
     bool is_full_frame;
-    mxfUL color_primaries;
-    mxfUL transfer_characteristic;
-    mxfUL matrix_coefficients;
+    mxfUL color_primaries[2];
+    mxfUL transfer_characteristic[4];
+    mxfUL matrix_coefficients[2];
 
     // guesses
-    uint8_t signal_standard;
+    uint8_t signal_standard;  // Can be changed to "none" if the colorimetry values don't match
     Rational aspect_ratio;
     mxfVideoLineMap video_line_map;
     uint8_t color_siting;
 } DefaultParameterMatch;
 
-// TODO: add more defaults
-// TODO: are the SD NTSC video line numbers correct?
+// TODO: add more guesses
 static const DefaultParameterMatch DEFAULT_PARAM_MATCHES[] =
 {
-    {720, 480, false, SMPTE170M_COLOR_PRIM, ITUR_BT709_TRANSFER_CH, ITUR_BT601_CODING_EQ,
+    {720, 480, false,
```

Comment:
I've added https://github.com/bbc/bmx/pull/32/commits/1d97306d56462386f7cdcc4385fc8d8949250e30 to fix the 525-line (NTSC) case and added https://github.com/bbc/bmx/pull/32/commits/9bbf9e933276f75fbdad82bc0f9f5354bdf3c2e8 for the 625-line (PAL) case.

Reply:
Updated the 625-line case in https://github.com/bbc/bmx/pull/32/commits/ca1a652ae901301604789a1ab57aa2ea65402cbf to fix the VideoLineMap to be `{23, 336}` as it was originally (not the DV one)

Reply:
I finally settled on `{20, 283}` given your Quicktime and [ffmpeg mxf2enc.c](https://github.com/FFmpeg/FFmpeg/blob/398467f519b3b4e954ac9e6868358c052b407fc5/libavformat/mxfenc.c#L1305) references as well as SMPTE ST 386M-2004 (D-10) which as a VideoLineMap `{7, 270}` with a DisplayYOffset of `13`.

Reply:
Another aspect to consider is that is should probably have to be `{21, 283}` if the bottom field is first in temporal order. D-10 has top field first which is why it is at 20 rather than 21.